analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -908,3 +1001,58 @@ struct StructWithValuePropertyAndCastOperators
     }
 }
 
+namespace TypeWithValueStaticProperty
+{
+    class Test
+    {
+        void Basics()
+        {
+            _ = AClassWithStaticValueProperty.Value;                                // Compliant, not on nullable value type
+            _ = AClassWithStaticValueProperty.Value.Value;                          // Compliant
+            _ = AClassWithStaticValueProperty.Value.Value.InstanceProperty;         // Compliant
+            _ = AClassWithStaticValueProperty.Value.Value.InstanceProperty.Value;   // Compliant
+            _ = new AClassWithInstanceValueProperty().Value;                        // Compliant
+        }
+    }
+
+    class AClassWithStaticValueProperty
```

Comment:
That is quite a mouth full:
```suggestion
    class StaticValue
```